exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -165,41 +176,69 @@ def _translate_to_prometheus(
         metric_name += self._sanitize(metric.name)
 
         description = metric.description or ""
+
+        metric_family_id = "|".join(
+            [metric_name, description, "%".join(label_keys), metric.unit]
```

Comment:
It looks like we aren't adding the InstrumentationScope here. I also don't see it in the spec, but I would have thought it would be necessary for building this key

Reply:
I am only using the arguments used to create the metric family object here, Instrumentation scope is not one of them.